client/src/pages/Login.js

```diff
@@ -1,16 +1,21 @@
-import React from "react";
+import React, { useState } from "react";
 import styled from "styled-components";
 import { Button, Flex, WhiteSpace } from "antd-mobile";
 import { Link } from "react-router-dom";
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import { validateEmail } from '../utils/common.js';
```

Comment:
good to see you have added email validation, we can also use this for the Need Help and Offer Help components with email inputs